src/Phinx/Console/Command/Migrate.php

```diff
@@ -113,14 +113,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln('<comment>warning</comment> performing fake migrations');
         }
 
-        // run the migrations
-        $start = microtime(true);
-        if ($date !== null) {
-            $this->getManager()->migrateToDateTime($environment, new \DateTime($date), $fake);
-        } else {
-            $this->getManager()->migrate($environment, $version, $fake);
+        try {
+            // run the migrations
+            $start = microtime(true);
+            if ($date !== null) {
+                $this->getManager()->migrateToDateTime($environment, new \DateTime($date), $fake);
+            } else {
+                $this->getManager()->migrate($environment, $version, $fake);
+            }
+            $end = microtime(true);
+        } catch (\Throwable $e) {
```

Comment:
As Phinx is still compatible with [PHP v5.6](https://github.com/cakephp/phinx/blob/v0.10.6/composer.json#L28) which doesn't have a Throwable class, there should be catch for an Exception:

```suggestion
        } catch (\Exception $e) {
            $output->writeln('<error>' . $e->getMessage() . '</error>');
            return 1;
        } catch (\Throwable $e) {
```

Reply:
You are right, thanks.